### DISCUSSION TITLE
Ideas not yet implemented

### DISCUSSION THREAD

**Discussion by trix7777** [trix7777/m2strm#18] [Ideas]
- Option to `IgnoreUndefinedGroups`: (working on it right now)
   - "New groups added to uwgroup file #15" by Richardvdg30
- Check against `uwtitles.cfg` for unwanted titles.
- Filtering levels. Right now the filter is tuned only for N1 provider.
- `oldgroups.log`:
   - groups no longer found when parsing/option to delete automatically from `uwgroups.cfg` - uncommented or not (exists in `uwgCfgArray` but not in `allGroupsArray`)
- Option to send mail (GNU Mailutils/Linux) when errors found and/or when new groups found.
- Minimum size of downloaded M3U8-file to parse - adds an extra layer of validity.
- Move invalid M3U8-file to log-directory - for fault tracing, adding date+time.
- Check for proper port in M3U8-file, if not valid - replace with correct port.
- Other validity checks: Must contain certain VOD-group and/or certain tv-channels.
- Retry download if failed with user set interval/times.

**Comment by BaQs** [comment]
May I add:
- option to clean (based on regex or keywords ?) title name. For instance: "DV", "IMAX" etc. are sometimes not parsed properly.
